src/components/EpisodeCard/index.tsx

```diff
@@ -1,6 +1,48 @@
 import React from "react"
-import "./styles.scss"
+import { ReactComponent as StarIcon } from "../../assets/star.svg"
+import { ReactComponent as StarFilledIcon } from "../../assets/starFilled.svg"
+interface IEpisodeCardProps {
+  episode: string
+  date: string
+  title: string
+  description: string
+}
 
-export default function EpisodeCard() {
-  return <div>EpisodeCard</div>
+export default function EpisodeCard({
+  episode,
+  date,
+  title,
+  description,
+}: IEpisodeCardProps) {
+  const [isFavorited, setIsFavorited] = React.useState(false)
```

Comment:
I think this part needs to manage by its parent component. Because you will list them and you need to which are favourited and which are not.

Reply:
is it sufficent to just define a prop and set the state to that prop's value ?

Reply:
Rather than using local state here, have a props to get the value, and have another props to change the value. Because parent needs access to the value.
```
...
isFavorited: boolean;
handleChangeIsFavorited: () => void;
...
```